src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -3174,6 +3184,183 @@ paths:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/ContentStorageInfo'
+        '400':
+          description: |
+            Invalid parameter: nodeId is not a valid format, or is not a file
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/storage-info/{contentPropName}/archive':
+    post:
+      x-alfresco-since: "7.2.0"
+      tags:
+        - storage-info
+      summary: Request to send given content to acrhive
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.2.0 and newer versions.
+        It also requires at least one specific implementation of underlying functionality in Cloud Connector(s).
+
+        Request to send given content to acrhive.
+      operationId: requestArchiveContent
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/contentPropNameParam'
+      responses:
+        '200':
+          description: Successful response
+        '400':
+          description: |
+            Invalid parameter: nodeId is not a valid format, or is not a file
+                               property name of content is not valid
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/storage-info/{contentPropName}/archive-restore':
+    post:
+      x-alfresco-since: "7.2.0"
+      tags:
+        - storage-info
+      summary: Request to restore given content from acrhive
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.2.0 and newer versions.
+        It also requires at least one specific implementation of underlying functionality in Cloud Connector(s).
+
+        Request to restore given content from acrhive.
+      operationId: requestRestoreContentFromArchive
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/contentPropNameParam'
+        - name: restoreArchivedContentRequest
+          in: body
+          description: |
+            Restore request parameters.
+            At the moment there is one parameter being supported which is restore priority.
+            Body is not mandatory.
+            High restore priority request body example:
+            ```JSON
+            {
+              "restorePriority": "High"
+            }
+            ```
+            Normal restore priority request body example:
+            ```JSON
+            {
+              "restorePriority": "Normal"
+            }
+            ```
+          required: false
+          schema:
+            $ref: '#/definitions/RestoreArchivedContentRequest'
+      responses:
```

Comment:
Should we also get a 403 for users without write permission? (I can't remember what permissions are required on this method so apologies if this is wrong :smile: )

Reply:
Thanks I added 403 and 409 and some more description to 400 as well.